core/src/main/scala/org/apache/spark/storage/RDDInfo.scala

```diff
@@ -55,7 +55,7 @@ class RDDInfo(
 }
 
 private[spark] object RDDInfo {
-  private val callsiteForm = SparkEnv.get.conf.get(EVENT_LOG_CALLSITE_FORM)
+  private lazy val callsiteForm = SparkEnv.get.conf.get(EVENT_LOG_CALLSITE_FORM)
```

Comment:
Is this related to the problem?

Reply:
I ran the test locally and this causes error when initializing `RDDInfo`. Actually I think this should be lazy because it is not always needed.

Reply:
Thanks for explaining.